src/app/app.component.html

```diff
@@ -59,6 +59,7 @@
           [attr.aria-current]="myAccountsRouterLink.isActive ? 'page' : undefined"
           role="link"
         >
+          <bb-icon-ui class="bb-layout__horizontal-nav-item-icon" name="transactions" size="md"></bb-icon-ui>
```

Comment:
I would recommend to include more icons than just one in the app. Find places to put icons so the impact of changing them is more prominent

Reply:
The project is in the early stage and will be filled out with more data and more components with icons. When it's stable we will update the project so more icons are replaced